bundle/compliance/policy/audit/file/ensure_ownership.rego

```diff
@@ -0,0 +1,21 @@
+package compliance.policy.audit.file.ensure_ownership
+
+import data.compliance.lib.common
+import data.compliance.policy.data_adapters.file as data_adapter
+
+finding(owner_user, owner_group) = result {
+	user = data_adapter.owner_user
+	group = data_adapter.owner_group
+	rule_evaluation := common.file_ownership_match(user, group, owner_user, owner_group)
```

Comment:
file resource specific common functions can be moved to a new `file/common.rego`